core/src/main/java/org/dcsa/conformance/core/check/JsonAttribute.java

```diff
@@ -583,6 +583,9 @@ public static JsonContentMatchedValidation combineAndValidateAgainstDataset(
     return (nodeToValidate, contextPath) -> {
       var codeA = nodeToValidate.path(nameA).asText("");
       var codeB = nodeToValidate.path(nameB).asText("");
+      if (codeA.isEmpty() || codeB.isEmpty() ) {
+        return Set.of();
+      }
```

Comment:
I am not ok with this change. The logic of this method was to validate that the fields combined matched against a data set. This changes exempts some cases if a field is absent/empty, but I have code using this method where this new exception should not apply. In fact, I am not sure this exception should ever apply.